### ISSUE TITLE
Titles as links by default

### ISSUE THREAD

**Issue by twMat** [TiddlyWiki/TiddlyWiki5#4071]
I share Steve Shneiders [description](https://youtu.be/CtaOL7gx3wc?t=735) that people find DnD to be a natural and simple way to import something from one wiki to another.

For this reason I propose that the setting for displaying tiddler titles as links is by default active.

I can see how one would not want *blue* titles but they don't have to be, as exemplified in the sidebar and as exemplified on a million websites. (It's not the 90'ies anymore - neither buttons nor links have to look like 'that'.) In fact, this is part of the proposal: 

Make the title, in the tiddlers head, appear the same regardless if it is a link or not.

Clicking the tiddlers title of a half scrolled tiddler is also the simplest way to navigate the tiddler to viewport top.

Edit: The hover effect should be similar to the sidebar-links

**Comment by pmario** [comment]
I would follow the suggestion, if you add: 

 - For Sticky titles. 
 - The hover effect should be similar to the sidebar-links

I don't see "title-links", if Sticky-titles are switched off.

**Comment by twMat** [comment]
>For Sticky titles.

Exclusively? I don't understand what you mean.

>The hover effect should be similar to the sidebar-links

Sure. 

>I don't see "title-links", if Sticky-titles are switched off.

Your titles are not links with "titles as links" active - unless you're using sticky titles?

**Comment by morosanuae** [comment]
I would also like tiddler titles as links. The DnD operation for importing tiddlers would be much easier and intuitive IMO. With the possibility for turning this functionality off or those who don't like the idea, of course.
I also like the possibility for scrolling to top of the tiddler but I guess this is related with sticky titles only.

**Comment by twMat** [comment]
>I also like the possibility for scrolling to top of the tiddler but I guess this is related with sticky titles only.

No, it is not a "sticky matter" if you're talking about a tiddler where you only see its *upper part beneath another tid*. Clicking the title link will then move it up to viewport edge.

**Comment by morosanuae** [comment]
> No, it is not a "sticky matter" if you're talking about a tiddler where you only see its upper part beneath another tid. Clicking the title link will then move it up to viewport edge.

It's ok then. I didn't consider this.

**Comment by inmysocks** [comment]
I don't like this idea. Titles as links has always been a source of annoyance for me, particularly on touch screens.

I would much prefer a drag and drop handle icon as an option on the tiddler toolbar or a similar less intrusive solution.

**Comment by morosanuae** [comment]
@inmysocks this can be resolved easily with a configurable option. Let each user decide. I do like the idea. It's just a matter of preference in the end. But I think the functionality should be there because it brings obvious advantages.

**Comment by inmysocks** [comment]
I am not arguing against the functionality, I just prefer that it isn't the default.

**Comment by pmario** [comment]
I think it makes sense, if the title is sticky. .. So we can make a new option, which does both. sticky title + title links.

**Comment by morosanuae** [comment]
@pmario it does makes sense even for non-sticky titles. See @twMat previous post for the case where you can see a tiddler upper part beneath another tiddler in the story river.

**Comment by twMat** [comment]
The guiding question has to be "what brings the most benefit for **most** users?"

I say it is links by default because of DnD and to set focus on a tiddler. A user, perhaps especially newcomers who, as Steve notes, really like to DnD, are less probable to discover a setting buried in the Ctrlpanel.

**Comment by morosanuae** [comment]
> The guiding question has to be "what brings the most benefit for most users?"

Even if this functionality brings many benefits for most of the users, it still needs to be a configurable option, and maybe not activated as default. I think we don't want to force anyone to accept it. Maybe what I'm saying it's obvious but I want to be sure.

**Comment by morosanuae** [comment]
But anyway I think @Jermolene will decide about this in the end.

**Comment by pmario** [comment]
>...I say it is links by default because of DnD and to set focus on a tiddler.

D&D is a strong argument
